### PR TITLE
[frontend] feature: add search input in in the admin panel of a service  (#1329)

### PR DESCRIPTION
**Pull request by Antoine-Quintin**
# Context
> Creating a search input in the admin panel of a service in order to search organizations name

## Related issues

- #1329 

# Checklist

## Quality

- [x] I consider this work complete and ready for review
- [x] I tested all features and edge cases
- [x] I refactored code where necessary to improve quality
- [x] I made sure my code meets development guidelines
- [x] I performed a self-review of my code

## Testing

- [ ] I wrote test cases for the relevant use cases (unit, integration or e2e tests)
- [ ] (Bug fixes only) I added tests that reproduce and verify the fix

### What tests have been made

- [x] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [x] Local manual tests

# Additional information
> Any additional context, dependencies, or concerns reviewers should know about.

### CODE REVIEW

File: apps/portal-front/messages/fr.json [hunk 1]
```diff
@@ -811,6 +811,7 @@
       "RemoveAccess": "Retirer les accès",
       "RemoveRights": "Supprimer les droits",
       "ShowPersonalSpaces": "Montrer les espaces personnels",
+      "SearchOrganization": "Organisation de la recherche",
```
Comment (jbanety) [inline apps/portal-front/messages/fr.json]: Hummm. Automagic French Translator here 😁 

```suggestion
      "SearchOrganization": "Rechercher une organisation",
```

**Review by jbanety — COMMENTED** [review]
Please update the PR description to explain the issue and your changes.

File: apps/portal-front/src/components/service/[slug]/service-slug.tsx [hunk 2]
```diff
@@ -223,14 +242,33 @@ const ServiceSlug: FunctionComponent<ServiceSlugProps> = ({
...
+          const filteredBySearch = searchTerm
+            ? filteredBySpace.filter((subscription) => {
+                const orgName = subscription?.organization?.name;
+                return orgName
+                  ?.toLowerCase()
+                  .includes(searchTerm.toLowerCase());
+              })
+            : filteredBySpace;
+
+          const sorted = [...filteredBySearch].sort((a, b) => {
```
Comment (Kroustille) [inline apps/portal-front/src/components/service/[slug]/service-slug.tsx]:
```suggestion
          const sorted = filteredBySearch.sort((a, b) => {
```

You can directly use the `filteredBySearch` array because sort returns a new array and does not modify `filteredBySearch`

**Review by Kroustille — APPROVED** [review]
